pkg/crc/oc/oc_cache.go

```diff
@@ -98,18 +98,18 @@ func (oc *OcCached) cacheOc() error {
 
 		// Copy the requested asset into its final destination
 		outputPath := constants.CrcBinDir
-		err = os.MkdirAll(outputPath, 0755)
+		err = os.MkdirAll(outputPath, 0750)
```

Comment:
Could this be 0700 throughout this commit so that it's consisten with the 0600/0400 change which is done for the files in a later commit?

Reply:
Sure, I didn't add it `700` because initial permission was `755`, just went safer side to overcome on those directory permission errors.